integration_tests/testsuite/deploy_app.py

```diff
@@ -43,7 +51,26 @@ def _set_builder_image(builder):
                 fout.write(line.replace('${STAGING_BUILDER_IMAGE}', builder))
 
 
-def deploy_app(base_image, builder_image, appdir, yaml):
+def _record_latency_to_bigquery(deploy_latency, language):
+    current_date = datetime.datetime.now()
+    row = [(language, current_date, deploy_latency)]
+
+    project = os.environ.get(DEPLOY_LATENCY_PROJECT_ENV)
+    client = bigquery.Client(project=project)
+    dataset = client.dataset(DATASET_NAME)
+    table = bigquery.Table(name=TABLE_NAME, dataset=dataset)
+    table.reload()
+    error = table.insert_data(row)
+    return error
+
+
+def deploy_app(
```

Comment:
nit: can you try and fit as many of these params on one line as possible? It's a little strange to look at with all of them on individual lines :) something like

```
def deploy_app(base_image, builder_image, appdir,
               yaml, record_latency, language):
```